demisto_sdk/commands/setup_env/configure_integration_in_server.py

```diff
@@ -0,0 +1,322 @@
+def __disable_integrations_instances(client, module_instances):
+    for configured_instance in module_instances:
+        # tested with POSTMAN, this is the minimum required fields for the request.
+        module_instance = {
+            key: configured_instance[key]
+            for key in [
+                "id",
+                "brand",
+                "name",
+                "data",
+                "isIntegrationScript",
+            ]
+        }
+        module_instance["enable"] = "false"
+        module_instance["version"] = -1
+        logger.debug(f'Disabling integration {module_instance.get("name")}')
+        try:
+            res = demisto_client.generic_request_func(
+                self=client,
+                method="PUT",
+                path="/settings/integration",
+                body=module_instance,
+            )
+        except ApiException:
+            logger.exception("Failed to disable integration instance")
+            return
+
+        if res[1] != 200:
+            logger.error(f"disable instance failed, Error: {pformat(res)}")
+
+
+# return instance name if succeed, None otherwise
+
+
+def create_integration_instance(
+    integration_name,
+    integration_instance_name,
+    integration_params,
+    is_byoi,
+    validate_test=True,
+):
+    failure_message = ""
+    # get configuration config (used for later rest api
+    integration_conf_client = demisto_client.configure()
+    configuration = __get_integration_config(integration_conf_client, integration_name)
+    if not configuration:
+        return None, "No configuration"
+
+    module_configuration = configuration["configuration"]
+    if not module_configuration:
+        module_configuration = []
+
+    __delete_integration_instance_if_determined_by_name(
+        integration_conf_client, integration_instance_name
+    )
+
+    logger.info(
+        f'Configuring instance for {integration_name} (instance name: {integration_instance_name}, validate "Test": {validate_test})'
+    )
+    # define module instance
+    module_instance = {
+        "brand": configuration["name"],
+        "category": configuration["category"],
+        "configuration": configuration,
+        "data": [],
+        "enabled": "true",
+        "engine": "",
+        "id": "",
+        "isIntegrationScript": is_byoi,
+        "name": integration_instance_name,
+        "passwordProtected": False,
+        "version": 0,
+    }
+    # set module params
+    for param_conf in module_configuration:
+        if (
+            param_conf["display"] in integration_params
+            or param_conf["name"] in integration_params
+        ):
+            # param defined in conf
+            key = (
+                param_conf["display"]
+                if param_conf["display"] in integration_params
+                else param_conf["name"]
+            )
+            if key == "credentials":
+                credentials = integration_params[key]
+                param_value = {
```

Comment:
I think there may be other options other than identifier/password, such as a certificate